android-sdk/android/src/main/res/values/logto.xml

```diff
@@ -1,4 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <string name="logto_redirect_scheme">logto_redirect_scheme_placeholder</string>
+    <string name="logto_redirect_scheme" />
+    <string name="wechat_app_id" />
+    <string name="alipay_app_id" />
```

Comment:
whats this?

Reply:
this should read from webview postmessage instead of defining here.

Reply:
These are placeholders for our sdk, we need these when implementing our sdk.
This config in the application will override these placeholders.
We don't have a convention about the way we use the app id?